src/storage/client/StorageClient.cpp

```diff
@@ -377,5 +377,11 @@ PartitionID StorageClient::partId(GraphSpaceID spaceId, int64_t id) const {
     return s;
 }
 
+int64_t StorageClient::getUUID(GraphSpaceID space, const std::string& field) {
+    LOG(INFO) << "Called GetUUID";
+    UNUSED(space);
+    return static_cast<int64_t>(std::hash<std::string>()(field));;
```

Comment:
Is this a temporary solution?

Reply:
yep this is rely on [Support uuid in storage service](https://github.com/vesoft-inc/nebula/pull/958)